.travis.yml

```diff
@@ -20,8 +20,8 @@ branches:
     - 8.x-1.x
 
 before_install:
-  - nvm install 6.11.0
-  - nvm use 6.11.0
+  - nvm install 6.11.1
```

Comment:
That won't work, because `.nvmrc` is only read for node_js projects. See https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions-using-.nvmrc